src/main/java/com/ericsson/ei/handlers/EventToObjectMapHandler.java

```diff
@@ -148,7 +154,7 @@ public boolean deleteEventObjectMap(String templateName) {
     }
 
     public boolean isEventInEventObjectMap(String eventId) {
-        String condition = "{\"_id\" : \"" + eventId + "\"}";
+        String condition = "{\"objects\": { \"$in\" : [\"" + eventId + "\"]} }";
```

Comment:
At the top of this class a variable is defined for this key called 'listPropertyName' - use it here also. However, given that you've changed the structure of this document to look like:

    ```
    "_id": "aggregated-object-id",
     "objects": ["event-id", "event-id"]
    ```
the key of the list should perhaps be called "events" instead of "objects"? It would look somehting like this instead:

    ```
    "_id": "aggregated-object-id",
     "events": ["event-id", "event-id"]
    ```

Reply:
> At the top of this class a variable is defined for this key called 'listPropertyName' - use it here also. However, given that you've changed the structure of this document to look like:
> 
> ```
> ```
> "_id": "aggregated-object-id",
>  "objects": ["event-id", "event-id"]
> ```
> ```
> 
> the key of the list should perhaps be called "events" instead of "objects"? It would look somehting like this instead:
> 
> ```
> ```
> "_id": "aggregated-object-id",
>  "events": ["event-id", "event-id"]
> ```
> ```

changed "events" instead of "objects"

Reply:
This "events" instead of "objects" changes will come in future, this changes will be release ASAP, could you please review the current changes

Reply:
I see another PR open, which duplicates this one. Please update the changes in this PR and close the other one (then we keep the review history also) It's possible for @SantoshNC68  to allow @durga-vasaadi as collaborator on your fork, and this way you can both push to this one PR.

Reply:
Yes I have done that and we have worked together, but because of the issues on the travis we have another  PR. We will close that.

Reply:
Also we will not make changes for events in this PR but will do it another PR as we need to see the impact of that.